### PR TITLE
Update Erigon runtime to new runtime API and Erigon

### PR DESCRIPTION
**Pull request by pnowosie**
This PR resolves following JIRA ticket

 
- [APPS-152](https://golemproject.atlassian.net/browse/APPS-152)

### Why
- We want to avoid problems with updating existing deployments in the future.
- We want nicer tutorial/example on how to implement runtimes.

### What
Update ya-runtime-erigon to use:
- Newest Erigon release (v2021.05.04-alpha)
- Latest yagna runtime SDK ([master-5d857c5, 2021-06-09](https://github.com/golemfactory/ya-runtime-sdk/commit/5d857c5e2fccb5b28a89b5b6fcf78d4c34649c5f))
- Reduce clutter from function responses

## Testing instructions

Try it in the debugger with supported [network values](https://github.com/golemfactory/yagna-service-erigon/blob/3957a286b7992cc6f181c7bacee94ed3b899c0a0/ya-runtime-erigon/src/main.rs#L35-L38)
```bash
ya-runtime-dbg \
  --runtime ./target/debug/ya-runtime-erigon \
  --workdir ./target/debug \
  --start-arg '{"network": "rinkeby"}'
```
image

### Assumptions 

- Erigon binaries placed in the same directory where the `ya-runtime-erigon` is located

### Response structure returned from `run_command`
Returned response json was cleaned from junk/example text.

```json
{
    "auth": {
        "password": "3gWsNVW1UT7Rvlj",
        "user": "erigolem"
    },
    "network": "rinkeby",
    "url": "http://erigon.localhost:8545"
}
```

### CODE REVIEW

**Review by mfranciszkiewicz — APPROVED** [review]

File: ya-runtime-erigon/Cargo.toml [hunk 1]
```diff
@@ -24,4 +24,4 @@ titlecase = "1.1.0"
 tokio = { version = "0.2", features = ["io-std", "io-util", "rt-core", "rt-threaded", "macros", "process", "time"] }
 
 [patch.crates-io]
-ya-runtime-sdk = { git = "https://github.com/golemfactory/ya-runtime-sdk.git", rev = "880efaf19111e71647dea23fd997ebf6edc6e3d7" }
+ya-runtime-sdk = { git = "https://github.com/golemfactory/ya-runtime-sdk.git", rev = "5d857c5e2fccb5b28a89b5b6fcf78d4c34649c5f" }
```
Comment (Wiezzel) [inline ya-runtime-erigon/Cargo.toml:27]: @mfranciszkiewicz Maybe we can have a release (tagged version)?

**Review by Wiezzel — APPROVED** [review]

File: ya-runtime-erigon/Cargo.toml (hunk 1 above)
Comment (mfranciszkiewicz) [reply ya-runtime-erigon/Cargo.toml:27]: @pnowosie @Wiezzel I've released `ya-runtime-sdk` to crates.io

Comment (Wiezzel) [inline ya-runtime-erigon/Cargo.toml:27]:
```suggestion
ya-runtime-sdk = "0.1.0"
```

**Review by Wiezzel — CHANGES_REQUESTED** [review]

File: ya-runtime-erigon/Cargo.toml (hunk 1 above)
Comment (pnowosie) [reply ya-runtime-erigon/Cargo.toml:27]: Good catch! It's time to switch to crates.

Comment (pnowosie) [reply ya-runtime-erigon/Cargo.toml]: BTW @mfranciszkiewicz @Wiezzel switching to crates.io breaks the build (now)
```
    Updating crates.io index
error: failed to resolve patches for `https://github.com/rust-lang/crates.io-index`

Caused by:
  patch for `ya-runtime-sdk` in `https://github.com/rust-lang/crates.io-index` points to the same source, but patches must point to different sources
Command \'cargo build\' has failed!
```
(checked on mac & ubuntu)

File: ya-runtime-erigon/Cargo.toml [hunk 2]
```diff
@@ -24,4 +24,4 @@ titlecase = "1.1.0"
 tokio = { version = "0.2", features = ["io-std", "io-util", "rt-core", "rt-threaded", "macros", "process", "time"] }
 
 [patch.crates-io]
-ya-runtime-sdk = { git = "https://github.com/golemfactory/ya-runtime-sdk.git", rev = "880efaf19111e71647dea23fd997ebf6edc6e3d7" }
+ya-runtime-sdk = "0.1.0"
```
Comment (mfranciszkiewicz) [inline ya-runtime-erigon/Cargo.toml:27]: Patching is no longer necessary

Comment (pnowosie) [reply ya-runtime-erigon/Cargo.toml]: Thanks! Resolved in [[#pull/23/commits/8816394](https://github.com/golemfactory/yagna-service-erigon/pull/23/commits/8816394ccbd6a103ed41a9afc59e4773c4750eb6)]

**Review by Wiezzel — APPROVED** [review]
:100:

**Pull request merged**